SabreTools.Models/SecuROM/MatroschkaFileEntry.cs

```diff
@@ -0,0 +1,64 @@
+namespace SabreTools.Models.SecuROM
+{
+    /// <summary>
+    /// File entry
+    /// </summary>
+    public sealed class MatroschkaFileEntry
+    {
+
+        /// <summary>
+        /// Value of unknown purpose. Seems to always be set to 1 for the first file (which seems to always be the 
+        /// executable that manages matroschka operations, i.e. the matroschka helper executable), 2 for the second file 
+        /// (which seems to always be the "main" executable, i.e. the RC-encrypted exe to be decrypted, the main game 
+        /// program executable, or the revoker executable), 4 for the third file (seems to always be a "helper"
+        /// executable or dll for the second file, such as DFA.dll for RC-encrypted executables since DFA decryption
+        /// will be needed, paul.dll for PA-protected games, or remover.exe for revocation executables.
+        /// My assumption is this has something to do with load order, but that's purely a guess.
+        /// </summary>
+        public int UnknownValueOne { get; set; }
+        
+        /// <summary>
+        /// Size
+        /// </summary>
+        public int Size { get; set; }
+
+        /// <summary>
+        /// Offset
+        /// </summary>
+        public int Offset { get; set; }
+
+        /// <summary>
+        /// Later matroschka sections have 4 0x00 bytes here, earlier ones don't.
+        /// </summary>
+        public int DummyOne { get; set; }
```

Comment:
If there are different revisions of section types, you can handle this 2 ways:
- Have one model per section type, usually with a chain of inheritance to make putting a list of them in the model easier. For example, if V2 always has extra fields over V1, then V2 would inherit from V1 and only include the extra values. This can get more complicated if they're intertwined, since it may mean just an empty base type that both V1 and V2 inherit from.
- If the differences are too intertwined, as is implied here, put a `<remarks>` below the summary and say which versions it's valid for. If you go this route, then the value should be nullable (`uint?`) to indicate it's not set, rather than set to a value of 0.